test/tools/libtesteth/TestHelper.h

```diff
@@ -112,7 +112,7 @@ json_spirit::mObject fillJsonWithState(eth::State const& _state, eth::AccountMas
 json_spirit::mObject fillJsonWithTransaction(eth::Transaction const& _txn);
 
 //Fill Test Functions
-int createRandomTest();	//returns 0 if succeed, 1 if there was an error;
+bool createRandomTest();	//returns 0 if succeed, 1 if there was an error;
```

Comment:
Update the comment, and everywhere that uses the return value.

Reply:
what should be updated everywhere that uses the return value?

Reply:
This function is used somewhere, no?

Reply:
Before, `0` was success and `1` was failure.  Now, `false` is failure and `true` is success.

Remember in C++, `if (false)` and `if(0)` is very similar.  So, something is probably broken.

Reply:
this one is only used in create random code. I always check that --createRandomCode option is working

Reply:
Does it check the return value of `createRandomTest()`?

Reply:
hm. that changes are in my local repo. can't link yet.